packages/@aws-cdk/aws-appmesh/lib/virtual-router-listener.ts

```diff
@@ -0,0 +1,122 @@
+import * as cdk from '@aws-cdk/core';
+import { CfnVirtualRouter } from './appmesh.generated';
+import { Protocol } from './shared-interfaces';
+
+/**
+ * Represents the properties needed to define Listeners for a VirtualRouter
+ */
+export interface RouterListenerProps {
```

Comment:
These should be `*Options`, so `RouterListenerOptions`. `*Props` are reserved for construction properties of Constructs. Apologies I did not notice this before (we'll probably have to go back and change a few of these in other places in the AppMesh module).

Reply:
No problem. Should updating other places be done in this in this PR or separate?

Reply:
Let's keep it separate.